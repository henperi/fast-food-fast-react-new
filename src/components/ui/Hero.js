import React from "react";

export const Hero = () => (
  <div className="hero">
    <div className="contain text-center">
      <h1 className="">Welcome To Fast Food Fast</h1>
      <p>
        The best food ordering app. Order afordable food items, drinks, pastries
        and African dishes to your door step instantly!
      </p>
      <form className="form-data" method="GET">
        <div className="form-member">
          <input
            type="text"
            name="search"
            className="form-input transparent text-center hero-input"
            placeholder="Search foods and dishes"
          />
          <a>
            <button name="search" className="btn btn-primary">
              Search Food
            </button>
          </a>
          <a href="foods-menu.html" className="btn btn-blue">
            Our Menu
          </a>
        </div>
      </form>
    </div>
  </div>
);
