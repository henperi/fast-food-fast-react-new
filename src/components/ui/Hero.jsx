import React from 'react';

import { NavLink } from 'react-router-dom';

const Hero = () => (
  <div className="hero">
    <div className="contain text-center">
      <h1 className="">Welcome To Fast Food Fast</h1>
      <p>
        The best food ordering app. Order afordable food items, drinks, pastries and African dishes
        to your door step instantly!
      </p>
      <form className="form-data" method="GET">
        <div className="form-member">
          <input
            type="text"
            name="search"
            className="form-input transparent text-center hero-input"
            placeholder="Search foods and dishes"
          />
          <button type="button" name="search" className="btn btn-primary">
            Search Food
          </button>
          <NavLink to="/users/foods" className="btn btn-blue">
            Our Menu
          </NavLink>
        </div>
      </form>
    </div>
  </div>
);

export default Hero;
