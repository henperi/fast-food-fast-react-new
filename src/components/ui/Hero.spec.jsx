import React from 'react';
import { shallow } from 'enzyme';
import Hero from './Hero';

test('should render Hero correctly', () => {
  const wrapper = shallow(<Hero />);
  expect(wrapper).toMatchSnapshot();
});
