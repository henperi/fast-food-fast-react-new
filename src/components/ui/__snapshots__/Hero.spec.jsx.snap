// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Hero correctly 1`] = `
<div
  className="hero"
>
  <div
    className="contain text-center"
  >
    <h1
      className=""
    >
      Welcome To Fast Food Fast
    </h1>
    <p>
      The best food ordering app. Order afordable food items, drinks, pastries and African dishes to your door step instantly!
    </p>
    <form
      className="form-data"
      method="GET"
    >
      <div
        className="form-member"
      >
        <input
          className="form-input transparent text-center hero-input"
          name="search"
          placeholder="Search foods and dishes"
          type="text"
        />
        <button
          className="btn btn-primary"
          name="search"
          type="button"
        >
          Search Food
        </button>
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="btn btn-blue"
          to="/foods-menu"
        >
          Our Menu
        </NavLink>
      </div>
    </form>
  </div>
</div>
`;
